00-HTML-CSS-basics/HTMLExercises/singUp.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Sign up form</title>
+</head>
+<body>
+<header>
+	<h1>Sign up</h1>
+	<h2>Please fill out this form</h2>
+</header>
+	<form action="#" method="post">
+		<p>
+        <label for="firstname"> First name: </label>
+          <input type="text" name="firstName" id="firstName" pattern="[A-Za-z]+" title="Letters only, please" required>
+          <small id="firstNameHelp"></small>
+        </p>
+        <p>
+	  	<label for="lastname"> Last name: </label>
+          <input type="text" name="lastName" id="lastName" pattern="[A-Za-z]+" title="Letters only, please" required>
+          <small id="lastNameHelp"></small>
+        </p>
+        <p>
+	  	<label for="email">E-mail:</label>
+          <input type="email" name="email" id="email" pattern="\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,4})+$" title="xxx@xxx.com" required>
```

Comment:
The input type email has some native checking, for this exercise is enough with that :)